packages/dht/src/dht/RouterDuplicateDetector.ts

```diff
@@ -0,0 +1,45 @@
+import { BloomFilter } from 'bloomfilter'
+
+// Link for calculating false positive probabilities
+// https://www.di-mgt.com.au/bloom-calculator.html
+export class RouterDuplicateDetector {
```

Comment:
Is this a generic duplicate detector for strings? If that is the case, we could rename this to `DuplicateDetector`.